pkgs/tools/filesystems/squashfs/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv, fetchFromGitHub, zlib, xz
 , lz4 ? null
```

Comment:
While we are at it can we also remove this syntactic sugar? Also in line 7.

Reply:
If this was called from a Nixpkgs without (a working) lz4, it'd end the eval because of a missing argument if no default was supplied.

Since that's a very unlikely scenario, we could just assume lz4 is always there and force lz4Support because I can't think of much of a reason to want to build it without either (we don't have anything like proper USE-flags yet).

Reply:
> without (a working) lz4

Nix complains if it is broken or not available on your platform.

I talked about this recently and it makes no difference code wise.